overlay.d/99okd/usr/lib/systemd/system-preset/00-okd.preset

```diff
@@ -1,10 +0,0 @@
-# Fetch GCP hostnames via afterburn
-enable gcp-hostnames.service
```

Comment:
We probably want to keep this file, no?

Reply:
I don't think it does anything. In previous commits I added a few services there and they were not enabled - only when I did 'systemctl enable' in the Dockerfile it worked.

Not sure why presets are no longer being respected